tox.ini

```diff
@@ -60,9 +60,9 @@ commands =
 [testenv:ui-tests]
 commands =
     make -f Makefile-docker update_deps
-    pip install --no-deps -r requirements/uitests.txt
     make -f Makefile-docker ui-tests
-    pytest --driver Remote --host selenium-firefox --port 4444 --capability browserName firefox --variables tests/ui/variables.json --self-contained-html tests/ui {posargs}
+    pip install --no-deps -r requirements/uitests.txt
```

Comment:
I wonder if this should be part of the `ui-tests` command similar to like everything dependency related is part of `update_deps`

Reply:
Well that is within the ```ui-tests``` tox command and that is what is run when you want to run the ui-tests.

Reply:
I mean to put that `pip install` line in the `ui-tests` make action so that the dependencies get automatically updated before ui-tests. Not a blocker though

Reply:
Ah, I like that idea.